src/editor/pivot_bounds_handle.cpp

```
@@ -0,0 +1,182 @@
+#include "bounds_widget.h"
+#include "editor_layer.h"
+#include "panels/editor_panel_canvas.h"
+#include "actions/editor_action.h"
+#include "moth_ui/events/event_dispatch.h"
+#include "moth_ui/layout/layout_entity.h"
+#include "moth_ui/nodes/node.h"
+#include "moth_ui/utils/transform.h"
+
+#include <cmath>
+
+PivotBoundsHandle::PivotBoundsHandle(BoundsWidget& widget)
```

Clang (14.0.6)

[error] constructor does not initialize these fields: m_originalPivot, m_originalOffset

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)